docs/publishing-repo.md

```diff
@@ -58,8 +50,9 @@ Consider [obtaining a DOI for your releases](https://guides.github.com/activitie
 
 
 ## Make your repo public
-If your repository is *private*, ask [tech support](how-to-get-help.md) to make it *public* before the associated paper or output is published.
-There's more information about what this means [here](repositories.md#repository-visibility).
+If your repository is *private*, click the "Change repo visibility" button on any workspace connected to it.
```

Comment:
Should mention that this is done in Job Server. The rest of the page refers to the git repo or GitHub.